<?xml version="1.0"?>
<launch>
	<!--include file="$(find alpha_drivers)/launch/drivers.launch"/-->
	<node pkg="alpha_localization" name="ahrs_node" type="ahrs_node" respawn="true" output="screen"/>
	<node pkg="alpha_localization" name="simple_pose_node" type="simple_pose_node" respawn="true" output="screen"/>
	<node pkg="nodelet" type="nodelet" name="autopilot_nodelet" args="manager" output="screen"/>
	<node pkg="nodelet" type="nodelet" name="AutoPilot" args="load alpha_autopilot/AutoPilot autopilot_nodelet" output="screen"/>
	<!--node pkg="alpha_autopilot" name="autopilot_node" type="alpha_autopilot_node" respawn="true" output="screen"-->

	<param name="autopilot_nodelet/roll/Kp" type="double" value="58.0"/>
	<param name="autopilot_nodelet/roll/Ki" type="double" value="0.3"/>
	<param name="autopilot_nodelet/roll/Kd" type="double" value="0.0"/>
	<param name="autopilot_nodelet/roll/upper_limit" type="double" value="4.100"/>
	<param name="autopilot_nodelet/roll/lower_limit" type="double" value="-4.100"/>
	<param name="autopilot_nodelet/roll/windup_limit" type="double" value="100.0"/>
	<param name="autopilot_nodelet/roll/cutoff_frequency" type="double" value="-1.0"/>

	<param name="autopilot_nodelet/pitch/Kp" type="double" value="0.54"/>
	<param name="autopilot_nodelet/pitch/Ki" type="double" value="0.1"/>
	<param name="autopilot_nodelet/pitch/Kd" type="double" value="0.0"/>
	<param name="autopilot_nodelet/pitch/upper_limit" type="double" value="4.100"/>
	<param name="autopilot_nodelet/pitch/lower_limit" type="double" value="-4.100"/>
	<param name="autopilot_nodelet/pitch/windup_limit" type="double" value="100.0"/>
	<param name="autopilot_nodelet/pitch/cutoff_frequency" type="double" value="-1.0"/>

	<param name="autopilot_nodelet/z/Kp" type="double" value="-0.32"/>
	<param name="autopilot_nodelet/z/Ki" type="double" value="-0.1"/>
	<param name="autopilot_nodelet/z/Kd" type="double" value="0.0"/>
	<param name="autopilot_nodelet/z/upper_limit" type="double" value="0.4"/>
	<param name="autopilot_nodelet/z/lower_limit" type="double" value="-0.4"/>
	<param name="autopilot_nodelet/z/windup_limit" type="double" value="1000.0"/>
	<param name="autopilot_nodelet/z/cutoff_frequency" type="double" value="-1.0"/>



</launch>
